### PR TITLE
Update openoffice.txt

### PR DESCRIPTION
**Pull request by Pilot-Pirx**
Update to version 4.1.10

### CODE REVIEW

File: openoffice.txt [hunk 1]
```diff
@@ -1,102 +1,100 @@
 [Section]
 Name = OpenOffice 4.1
-Version = 4.1.5
-License = LGPL
+Version = 4.1.10
+License = AL v2.0
 Description = Open source office suite.
-Size = 134.2 MiB
+Size = 135,98 MiB
```
Comment (learn-more) [inline openoffice.txt:6]: Size is not needed if SizeBytes is present.

Comment (Pilot-Pirx) [reply openoffice.txt:6]: OK, I just updated the old file...
SizeBytes is preferred?

File: openoffice.txt [hunk 2]
```diff
@@ -1,102 +1,89 @@
...
-SizeBytes = 128251698
+URLSite = https://www.openoffice.org/tr/
+URLDownload = https://download.sourceforge.net/project/openofficeorg.mirror/4.1.10/binaries/tr/Apache_OpenOffice_4.1.10_Win_x86_install_tr.exe
+SHA1 = f4555bcb48018032a151dd8ef068362b97dc341e
+SizeBytes = 129619608
 
 [Section.0422]
 Description = Відкритий офісний пакет.
-URLSite = http://www.openoffice.org/ua/
+URLSite = https://www.openoffice.org/uk/
```
Comment (learn-more) [inline openoffice.txt:75]: `ua` is not `uk`, is this an intentional change?

Comment (Pilot-Pirx) [reply openoffice.txt:75]: Yes, uk is the language code for Ukrainian. ua is the country code for the Ukraine.
OpenOffice only supports languages, not countries.
That was fixed on the homepage.

Comment (binarymaster) [reply openoffice.txt:75]: Confirming, `/ua/` leads to 404 page.

**Review by blackcrack — APPROVED** [review]

**Review by learn-more — APPROVED** [review]

**Comment by Pilot-Pirx** [comment]
"1 workflow awaiting approval
First-time contributors need a maintainer to approve running workflows"

Any idea when this can get merged?

**Pull request merged**